### PR TITLE
Use Alpine base image and install Ruby from source.

### PR DESCRIPTION
**Pull request by robinmitra**
This is to decouple the dependency between Alpine and Ruby versions.

Co-authored-by: Alan Carter <alan.carter@digital.cabinet-office.gov.uk>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,22 +1,16 @@
-# WARNING: We must pin the Alpine version to 3.8 because freeradius-rest (r6) package in alpine-3.9 contains
-# a regression breaking validation of rest authentication responses. Causes health checks to fail.
-# This is a known issue with FreeRadius: https://github.com/FreeRADIUS/freeradius-server/issues/2821
-FROM ruby:2.6.3-alpine3.8
+FROM govwifi-frontend-base:latest
```
Comment (sarahseewhy) [inline Dockerfile:1]: This is purely for my curiosity. Where and how do we get this image? I can see it's from Dockerfile.base. Do we need to update the documentation in the repo to reflect the new process?

**Review by sarahseewhy — APPROVED** [review]
Looks fine. I have one question about documenting the new process.

File: Dockerfile (hunk 1 above)
Comment (sarahseewhy) [reply Dockerfile:1]: Never mind, I found the other PR: https://github.com/alphagov/govwifi-concourse-deploy-pipeline/pull/14/files

**Comment by robinmitra** [comment]
Hey @sarahseewhy cheers for looking at this but this is being superseded by: https://github.com/alphagov/govwifi-frontend/pull/113

Apologies I forgot to close this one.

**Pull request closed without merging**